README.md

```diff
@@ -129,3 +129,5 @@ XRPL Bounties expresses sincere gratitude to all of the contributors who help to
 
 - See other people's ideas for bounties: https://github.com/XRPLBounties/Proposals/discussions
 - Share an idea for a bounty: https://github.com/XRPLBounties/Proposals/discussions/new?category=ideas
+
+## If you have questions, please send them to info@xrplbounties.org (We will aim to respond within 2-3 business days)
```

Comment:
```suggestion
## If you have questions, please send them to info@xrplbounties.org
```

I don't think we need to commit to a timeline here, but we should aim to respond within 3 business days regardless. What we claim here doesn't really matter - the important thing is to actually do it.

Reply:
Makes sense - accidentally merged without this change, then directly updated it to match this.